index.rst

```diff
@@ -6,20 +6,423 @@
+.. figure:: /_static/turbopump_setup.png
+   :name: turbopump-setup
+   :alt: Turbopump setup
+
+   Left - Connecting the turbopump to the dewar. Right - Vacuum hose connection to the dewar. 
+
+Remove the vacuum covers, and clean the turbopump and the hose flange with isoporpyl alcohol (using gloves), to avoid introducing contaminants inside the turbopump and hose.
+Connect the hose to the turbopump flange. 
+The hose has to be in a relaxed shape, with no turn or folds. 
+
+After checking the instrument dewar valve is closed, clean both the vacuum hose and instrument dewar flange with alcohol.
+Connect the other end of the hose to the dewar, without forcing the hose into a stressed position.
+This is why it may be necessary to rotate Nasmyth #2 between 0° and 20° to find a comfortable fit. 
+
+Check the turbopump output valve is closed.
+
+.. figure:: /_static/instrument_dewar_valve.png
+   :name: instrument-dewar-valve
+   :alt: LATISS dewar valve
+
+   Picture of the LATISS dewar valve. 
+
+.. note:: 
+   The turbopump pump output valve is not typically used during the procedure. 
+   What is important is that the pressure in the vacuum line to be lower than the pressure of the dewar BEFORE opening the dewar valve. 
+
+Starting the turbopump
+----------------------
+
+Connect the external pressure sensor to the turbopump. 
+This is an external sensor that is mounted on the top of the turbopump, and connects to an electronic display. 
+See figure 7.
+This pressure sensor is identical to the dewar pressure sensor on the first floor. 
+To start the electronic display read out, press and hold the up arrow key for three seconds.
+
+.. figure:: /_static/pressure_sensor.png
+   :name: pressure-sensor
+   :alt: Turbopump pressure sensor
+
+   Left – Turbopump vacuum line sensor (connected to the top of the pump). Right – Pressure electronic display.
+
 .. note::
+   The setup in the left photo of Fig. 7 is example only. 
+   Please do not operate the Turbopump on a non-rigid surface. 
 
-   **This technote is a work-in-progress.**
+Connect the turbopump to a secure electrical outlet, and turn the flip switch (at the back of the device on the lower right) to turn the pump on.
+This only puts the pump in operational condition, it’s not evacuating air yet.
 
-Abstract
-========
+Start the pump (with the front power button). 
+The turbopump will spin up to 1500 Hz in full operation.
+
+.. figure:: /_static/operational_turbopump.png
+   :name: operational-turbopump
+   :alt: Operational turbopump
+
+   Turbopump is fully spun-up.
+
+Before opening the instrument dewar valve, it’s necessary to have similar vacuums on the pump and inside the dewar. 
+If this is not the case, the turbopump or the dewar inside LATISS could receive a high pressure shock. 
+Ideally, you should wait until the pressure is lower on the turbopump's side.
+
+- If the dewar was evacuated completely, its pressure should be close to the turbopump at this point (~ atmospheric pressure).
```

Comment:
I think this is ambiguous... "evacuated completely" means (to me) that it'll be at a really high vacuum, not at atmospheric pressure.
If you want the gory details, consider something like:
"If the dewar has been warmed up but not opened, the pressure will be approximately the partial pressure of water vapour (~20 Torr). In the case the dewar has been serviced or opened, it may be at atmospheric pressure or under a slight positive pressure if purged with dry nitrogen."
If not, then just replace "evacuated completely" with "warmed up to ambient temperature" or something like that.

Reply:
👍